backend/dataall/modules/redshift_datasets/api/connections/resolvers.py

```diff
@@ -38,6 +38,31 @@ def list_redshift_schema_tables(context: Context, source, connectionUri: str, sc
     return RedshiftConnectionService.list_schema_tables(uri=connectionUri, schema=schema)
 
 
+def add_redshift_connection_group_permissions(
+    context: Context, source, connectionUri: str, groupUri: str, permissions: list
+):
+    RequestValidator.required_param('connectionUri', connectionUri)
+    RequestValidator.required_param('groupUri', groupUri)
+    RequestValidator.required_param('permissions', permissions)
+    return RedshiftConnectionService.add_group_permissions(uri=connectionUri, group=groupUri, permissions=permissions)
+
+
+def delete_redshift_connection_group_permissions(context: Context, source, connectionUri: str, groupUri: str):
+    RequestValidator.required_param('connectionUri', connectionUri)
+    RequestValidator.required_param('groupUri', groupUri)
+    return RedshiftConnectionService.delete_group_permissions(uri=connectionUri, group=groupUri)
+
+
+def list_redshift_connection_group_permissions(context: Context, source, connectionUri: str, filter: dict = None):
+    RequestValidator.required_param('connectionUri', connectionUri)
+    return RedshiftConnectionService.list_connection_group_permissions(uri=connectionUri, filter=filter)
+
+
+def list_redshift_connection_group_no_permissions(context: Context, source, connectionUri: str, filter: dict = None):
+    RequestValidator.required_param('connectionUri', connectionUri)
+    return RedshiftConnectionService.list_connection_group_no_permissions(uri=connectionUri, filter=filter)
```

Comment:
are those `required_param` checks necessary? don't we enforce the presence by setting `NoNullable` in the GQL schema?

Reply:
Not really, a better check would be the validity of those params. Like the length of the connectionUri or permissions. Lete me rework it a bit